New package: DiffMatic v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: DiffMatic
- Repository: https://github.com/asterycs/DiffMatic.jl
- Created by: @asterycs
- Version: v0.1.0
- Commit: a528414ece15bf191ef31d96b2dfb408e1575383
- Reviewed by: @asterycs
- Reference: https://github.com/asterycs/DiffMatic.jl/commit/a528414ece15bf191ef31d96b2dfb408e1575383#commitcomment-157328308
- Description: A library for differentiating vector/matrix/tensor expressions.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6d7f9dc38c36d8588070a1b0a1284a44de7efbec322df26278f2f32f5b0be5907e5dcc56a2b816a46f9ff7bb9a2b4d17a6fcf7e41ad8e2646475338858bdb65cc114d6d81ff833cf5e47f336c8492ad10c35c9245444878777498614f1ae39ddd3d65d26f069082ce69f309f3f340f5e92cea4e1a93fbde52e0305301ba4993f0b580f4eeaf33ea5d37a2a9cdb5b9fad670fdd2eaa8d8abaa21ac8ae5de4faeab700c28c2e13d9bde7e4cfb0381785460 -->

asterycs commented:
Let me just cancel this one - I'll make a new one after I've fixed an issue. Sorry for the noise!

JuliaRegistrator committed:
New package: DiffMatic v0.1.0

UUID: ec55c3ff-feb8-4051-a6a4-2e9f18c613fc
Repo: https://github.com/asterycs/DiffMatic.jl.git
Tree: 196d3d30b6ebf664b1e5377639ff15baf7f2bd47

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
